sdk/go/x/auto/minimum_version.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auto
+
+import "github.com/blang/semver"
+
+var minimumVersion = semver.Version{
```

Comment:
IS there anything we are going to need to do in our 3.0 work relating to this? Is this something we need to bump on each release?

Reply:
> Is there anything we are going to need to do in our 3.0 work relating to this?

Yes, for the 3.0 release we will need to bump this to 3.0.0. 

> Is this something we need to bump on each release?

No, only when there is a change to the automation SDK that is dependent on a CLI change.

Reply:
> Yes, for the 3.0 release we will need to bump this to 3.0.0.

Out of curiosity - is this true?  Will 3.0 of automation API require that a 3.0 CLI is installed?  It's not immediately obvious it would have to?

Related - is it a breaking change to increase this?  I assume so - which will make us want to do this very infrequently.

Reply:
Yes. We made breaking changes to the CLI in 3.0 (standardizing select behavior) that we use under the hood in Automation API, so we will need to bump the minimum version.